bin/cohere-embed-v3-services.ts

```diff
@@ -33,12 +34,13 @@ const cohereEmbedModel = process.env.COHERE_EMBED_MODEL!;
 const imageVersion = process.env.IMAGE_VERSION!;
 const ecrRepositoryName = process.env.ECR_REPOSITORY_NAME!;
 const dockerfileName = process.env.DOCKERFILE_NAME!;
-const port = process.env.PORT!;
+const cdkDeployPort = process.env.PORT!;
```

Comment:
**Suggestion:** Consider renaming the `cdkDeployPort` variable to `port` for consistency with the existing variable naming convention.
> 🍄  **Medium Priority** 
 > [category: **`Consistency`**, importance: **`6`**]
> **Reason:** Renaming the variable `cdkDeployPort` to `port` improves consistency with the existing variable naming convention, enhancing code readability and maintainability.

```suggestion
37 +const port = process.env.PORT!;
```